src/lib/Parser.ts

```diff
@@ -0,0 +1,59 @@
+import { ParserOptions, ParsedAst, FormatterOptions } from './types'
+import { Parser } from 'node-sql-parser/build/postgresql'
+import prettier from 'prettier/standalone'
+import SqlFormatter from 'prettier-plugin-sql'
+
+const DEFAULT_PARSER_OPTIONS = {
+  database: 'PostgreSQL',
+  type: 'table',
+}
+
+const DEFAULT_FORMATTER_OPTIONS = {
+  plugins: [SqlFormatter],
+  formatter: 'sql-formatter',
+  language: 'postgresql',
+  database: 'postgresql',
+  uppercase: false, // I thought this would "lowercase" everything, but it doesn't. There is not option to convert everything to lowercase.
+  parser: 'sql',
+}
+
+/**
+ * Parses a SQL string into an AST.
+ */
+export function Parse(sql: string, options: ParserOptions = {}): ParseReturnValues {
```

Comment:
Ideally the parser would also include the ability to replace patterns so that we can do templates (similar to Redash)